pyproject.toml

```diff
@@ -0,0 +1,89 @@
+[tool.poetry]
+name = "tg-react"
+version = "3.0.0b0"
+description = "Helpers for react based applications running on django."
+authors = ["Thorgate <jyrno@thorgate.eu>"]
+license = "BSD"
+homepage = "https://github.com/thorgate/tg-react"
+repository = "https://github.com/thorgate/tg-react"
+readme = "README.md"
+packages = [
+    { include = "tg_react" },
+]
+include = [
+    "tg_react/**/*.txt",
+    "tg_react/**/*.html",
+    "tg_react/**/*.mo",
+    "tg_react/**/*.po"
+]
+keywords = ["tg-react", "thorgate", "rest", "drf"]
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Framework :: Django',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: BSD License',
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+]
+
+[tool.poetry.dependencies]
+django = "django>=2.1.5,<4.0"
+djangorestframework = ">=3.6.4"
+
+python = "^3.6"
+model-bakery = "^1.2.1"
```

Comment:
I suppose these two as runtime deps will also get installed when installing the package? What about moving them to dev-deps?

Reply:
good catch, these should be under dev dependencies